includes/helper/class-wp-job-manager-helper.php

```diff
@@ -499,7 +597,7 @@ public function licence_output() {
 		if ( ! empty( $_POST ) ) {
 			$this->handle_request();
 		}
-		$licenced_plugins = $this->get_installed_plugins();
+		$licenced_plugins = $this->get_installed_plugins( false, false );
```

Comment:
I don't understand why we were limiting this to active only before.

Reply:
No idea either. Multisite scenario came to my mind, but not really sure there is any issue with showing this for the inactive too there.